### ISSUE TITLE
How can I use regexp identifier "\b" on every wild card?

### ISSUE THREAD

**Issue by Folkheart** [abo-abo/swiper#2647]
Is there a way to make a custom wild card when I use the space character in a search? I want to use something like "\b.*" when hitting space to look for every word beginning with my search input.
For example, I am searching in counsel-M-x for counsel-switch-buffer-other-window so I type "co sw bu" meaning I want all the matches that have one word beginnig with "co" followed with a word beginnig with "sw" and the next word starting with "bu".
It may work better reducing matching candidates when using ivy--regex-plus or ivy--regex-ignore-order.

**Comment by muirdm** [comment]
Yes, you can create your own regex builder similar to ivy--regex-plus.
